### DISCUSSION TITLE
App updates and Couchbase persistence

### DISCUSSION THREAD

**Discussion by tommag21** [triniwiz/nativescript-plugins#168] [Q&A]
Hi,

I developed an app using the Couchbase plugin which is currently in its 10th update on Play Store and App Store. My client informed me that after every update many users experience issues related to the data stored in the local Couchbase database, both on Android and iOS but mostly Android. The only fix is to clean the app cache, which forces the app to download the data anew on startup.

While I am looking for a bug in my code, I was wondering if the Couchbase database is persisted when the app is updated.

**Comment by triniwiz** [comment]
It should
